torch/csrc/dynamo/eval_frame.c

```diff
@@ -545,14 +562,13 @@ static PyObject* lookup(CacheEntry* e, THP_EVAL_API_FRAME_OBJECT *frame, CacheEn
     }
```

Comment:
You leak result here

Reply:
You're still leaking result. You need to deallocate it in the early return branch.